### ISSUE TITLE
repository not found in the readme.md

### ISSUE THREAD

**Issue by Samuel-Scalbert** [softcite/software-mentions#35]
Hello, I think the link for the git repository in the readme.md isn't working :

```sscalber@ss-inria:~/GRASPOS/grobid-0.8.0$ git clone https://github.com/kermitt2/software-mentions
Clonage dans 'software-mentions'...
Username for 'https://github.com': ...
Password for 'https://github.com': ...
remote: Repository not found.
fatal: dépôt 'https://github.com/kermitt2/software-mentions/' non trouvé```

**Comment by kermitt2** [comment]
Hello thanks ! Yes, typo, this is the current repository that has to be cloned. Corrected with 5c481b11fae1baadf9f4b9bee3f85ad5c4d75085

**Comment by Samuel-Scalbert** [comment]
Thnak you !
